homeassistant/components/rachio/switch.py

```diff
@@ -179,6 +185,72 @@ async def async_added_to_hass(self):
         )
 
 
+class RachioRainDelay(RachioSwitch):
+    """Representation of a rain delay status/switch."""
+
+    def __init__(self, controller):
+        """Instantiate a new Rachio rain delay switch."""
+        super().__init__(controller, poll=True)
+        self._poll_update(controller.init_data)
```

Comment:
We don't want side effects in init method. If we want a single update during entity addition we can use `async_add_entities(..., True)` for that, and implement `async_update` entity method.

Reply:
https://github.com/home-assistant/core/pull/34799